### DISCUSSION TITLE
Prefetching root "/" returns 404 error when running on Vercel (Nextjs 13.4.8)

### DISCUSSION THREAD

**Discussion by fabianszabo** [vercel/next.js#52339] [App Router]
Hello everybody! 👋🏼 

I get a 404 error when Nextjs tries to prefetch "/". All other paths are prefetching just fine.
<img width="348" alt="image" src="https://github.com/vercel/next.js/assets/85681889/5f90ce70-0ce5-4eb7-b633-8395d504bbbd">

Opening that page works just fine and there's nothing on that page that's not also on at least one other page.

Prefetching is triggered by next/link (i'm using it like this: `<Link href="/">Home</Link>`)

Adding `prefetch={false}` prevents that error.

I don't get the error when I run it locally, neither when using `next build` + `next start` nor `next dev`.
<img width="393" alt="image" src="https://github.com/vercel/next.js/assets/85681889/b81d6089-2f28-4110-8b6a-db19019ba69b">

Using Nextjs 13.4.8, running on Vercel

Please let me know if you need more details 🙏🏼 

Thank you in advance!

Fabian

**Comment by victormattosvm** [comment]
Same here with Next 14.2.29. Trying to debug. Did you manage to fix it?
